optimum/exporters/tasks.py

```diff
@@ -210,6 +223,22 @@ class TasksManager:
             "transformers": _TRANSFORMERS_TASKS_TO_MODEL_LOADERS,
         }
 
+        auto_modeling_module = importlib.import_module("transformers.models.auto.modeling_auto")
+        for task_name in _TRANSFORMERS_TASKS_TO_MODEL_LOADERS:
+            if isinstance(_TRANSFORMERS_TASKS_TO_MODEL_LOADERS[task_name], tuple):
+                _TRANSFORMERS_TASKS_TO_MODEL_MAPPINGS[task_name] = []
+                for model_loader_name in _TRANSFORMERS_TASKS_TO_MODEL_LOADERS[task_name]:
+                    model_loader_class = getattr(auto_modeling_module, model_loader_name, None)
+                    if model_loader_class is not None:
+                        _TRANSFORMERS_TASKS_TO_MODEL_MAPPINGS[task_name].append(
+                            model_loader_class._model_mapping._model_mapping
+                        )
+            elif isinstance(_TRANSFORMERS_TASKS_TO_MODEL_LOADERS[task_name], str):
+                model_loader_name = _TRANSFORMERS_TASKS_TO_MODEL_LOADERS[task_name]
+                model_loader_class = getattr(auto_modeling_module, model_loader_name, None)
+                if model_loader_class is not None:
+                    _TRANSFORMERS_TASKS_TO_MODEL_MAPPINGS[task_name] = model_loader_class._model_mapping._model_mapping
```

Comment:
I am not a big fan of defining class attributes like this, here I believe `auto_modeling_module`, `model_loader_class`, `model_loader_name`, etc. remain class attributes and are not treated as temporary variables.

Reply:
oh I see what you mean, I'll put this part in some function to limit the scope.